src/webparts/todolist/components/ToDoList.module.scss

```diff
@@ -10,86 +10,15 @@
   }
 }
 
-.welcome {
-  text-align: center;
+.ItemList {
+  border-radius: 16px;
+  height: 24px;
+  white-space: "nowrap";
+  padding: 4px 8px;
+  max-width: max-content;
 }
 
-.welcomeImage {
-  width: 100%;
-  max-width: 420px;
-}
-
-.links {
-  a {
-    text-decoration: none;
-    color: "[theme:link, default:#03787c]";
-    color: var(
-      --link
-    ); // note: CSS Custom Properties support is limited to modern browsers only
-
-    &:hover {
-      text-decoration: underline;
-      color: "[theme:linkHovered, default: #014446]";
-      color: var(
-        --linkHovered
-      ); // note: CSS Custom Properties support is limited to modern browsers only
-    }
-  }
-}
-
-.list {
-  color: #333333;
-  font-size: 14px;
-  font-weight: normal;
-  box-sizing: border-box;
-  margin: 10px;
-  padding: 10px;
-  line-height: 50px;
-  list-style-type: none;
-  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.2), 0 25px 50px 0 rgba(0, 0, 0, 0.1);
-}
-
-.listItem {
-  color: #333333;
-  vertical-align: center;
-  box-sizing: border-box;
-  box-shadow: none;
-  *zoom: 1;
-  padding: 9px 28px 3px;
-  position: relative;
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-}
-
-.itemStatus {
-  width: 100px;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  border-radius: 1rem;
-}
-
-.itemPending {
-  background-color: rgb(159, 159, 8);
-  color: white;
-}
-
-.itemCompleted {
-  background-color: rgb(40, 150, 40);
-  color: white;
-}
-
-.itemActive {
-  background-color: rgba(40, 40, 212, 0.84);
-  color: white;
-}
-
-.itemOverdue {
-  background-color: rgba(200, 74, 36, 0.84);
-  color: white;
-}
-
-.itemIcon {
-  margin-left: 4px;
+.statusPending {
+  background-color: "#ffe092";
```

Comment:
I would prefer to use on of the accent colors intead own one.
https://developer.microsoft.com/en-us/fluentui#/styles/web/colors/theme-slots

So, your color will be smth like `color: "[theme: yellow, default:#ffe092]"`